app/renderer/js/main.ts

```diff
@@ -961,6 +985,19 @@ class ServerManagerView {
 		ipcRenderer.on('new-server', () => {
 			this.openSettings('AddServer');
 		});
+
+		ipcRenderer.on('mute-org', (event: Event, index: number) => {
+			if (DomainUtil.getDomain(index).muted) {
+				// server is already muted
+				this.tabs[index].webview.updateBadgeCount(DomainUtil.getDomain(index).badgeCount);
+				DomainUtil.updateBadgeCount(index, -1);
+				DomainUtil.updateMute(index, false);
```

Comment:
Let's define a function that takes care of this update instead of writing the same code repeatedly?

Reply:
I have added the functions in the `main.ts` file, the functions are: `muteOrg()` and `unMuteOrg()`